README.md

```diff
@@ -88,6 +88,21 @@ FrederickAPI::V2::Location.with_access_token(access_token) do
 end
 ```
 
+### Using Headers
+
+`JsonApiClient::Resource` exposes a way to add headers that is written over in the `with_access_token` method.
+An example use case is adding an `X-Universal-Customer` header.
```

Comment:
Since the with_headers is written over already by with_access_token, I added the additional headers to the same method. That was what I was trying to get through. Ill write this better